src/components/sync-connected-sites.tsx

```diff
@@ -104,6 +111,7 @@ export function SyncConnectedSites( {
 								variant="link"
 								className="!ml-auto !text-a8c-gray-70 hover:!text-a8c-red-50 "
 								onClick={ () => handleDisconnectSite( section.id, section.name ) }
+								disabled={ isAnySitePulling }
```

Comment:
What is the reason for disabling the `Pull` button for all Studio sites when one of the Studio site is pulling?

Reply:
I disabled the buttons in the design as I assumed it would be more complex to handle multiple Push/Pull operations at the same time. Happy to lift the limitation if that's not the case though.

Reply:
I think it's a reasonable limitation. We could optimize it in the future, but if the user imports multiple sites at the same time we may need to lead with multiple threads, etc.

I tested the import flow to compare, and confirmed that even the UI is enabled for the rest of sites, the import flow doesn't work for the second site.

I've created an issue to disable the import UI for that case:
- https://github.com/Automattic/dotcom-forge/issues/9759